jina/flow/base.py

```diff
@@ -992,6 +981,7 @@ def rolling_update(self, pod_name):
         """
         for i, replica in enumerate(self._pod_nodes[pod_name].replica_list):
             replica.close()
+            time.sleep(1)
```

Comment:
why is the sleep needed?

Reply:
From time to time the test hangs. I thought it might be because some waiting between close and start is required. But that is not the reason. I will remove it.